pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -131,38 +131,112 @@ var _ = Describe("TKGS - Create workload cluster use cases", func() {
 		})
 		When("cluster class cli feature flag (features.global.package-based-lcm-beta) is set to true", func() {
 			BeforeEach(func() {
-				//set the cli feature flag as true -  (features.global.package-based-lcm-beta)
+				// set the cli feature flag as true -  (features.global.package-based-lcm-beta)
 				Expect(framework.SetCliConfigFlag(CLI_CLUSTERCLASS_FLAG, "true")).To(Succeed(), "error while setting CLI feature flag")
 				tkgctlClient, err = tkgctl.New(tkgctlOptions)
 				Expect(err).To(BeNil())
```

Comment:
All of thesse (line 104, 97, etc.. seem to be the exact same code. Any way to refactor it? i.e. define a function that gets called in BeforeEach, it will make future maintenance easier.